src/style.css

```diff
@@ -0,0 +1,46 @@
+body,
+div {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
`body` doesn't have default margin. `div` doesn't even have padding

Reply:
When I delete margin: 0  for body the stars I see this thing
![Screenshot_bodymargin](https://github.com/mate-academy/layout_stars/assets/49691082/5bb3c214-fbb7-4fde-afcc-2a08977febde)

Reply:
Delete div but body stay with margin: 0.